main.go

```diff
@@ -124,6 +124,10 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "KeystoneEndpoint")
 		os.Exit(1)
 	}
+	if err = (&keystonev1.KeystoneAPI{}).SetupWebhookWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create webhook", "webhook", "KeystoneAPI")
+		os.Exit(1)
+	}
```

Comment:
I think we should also add the self-signed cert `make` target like you did for the similar Glance webhook PR

Reply:
Yeah, I've not updated it waiting for the glance one to merge.

Reply:
Done!